lib/use-google-map.js

```diff
@@ -0,0 +1,170 @@
+// https://janosh.io/blog/google-maps+react-hooks, and own tweaks
```

Comment:
Since this is being used on the `host` app as well, I'm thinking I should add this either to some sort of universe utilities repo, or somewhere else... 🤔